policyengine_us/variables/gov/irs/credits/ctc/refundable/ctc_social_security_excess.py

```diff
@@ -0,0 +1,16 @@
+from policyengine_us.model_api import *
+
+
+class ctc_social_security_excess(Variable):
```

Comment:
Is this the right variable name? It's reused after reducing by EITC in the main formula.

Reply:
good point, i changed the name to follow the legal code as they refer to it as "social security taxes"